torch/_inductor/compile_fx.py

```diff
@@ -346,11 +346,6 @@ def compile_fx(
     inner_compile=compile_fx_inner,
 ):
     """Main entrypoint to a compile given FX graph"""
-
-    if not is_aot_autograd_safe_to_run(model_, example_inputs_):
-        log.warning("Aot Autograd is not safe to run, so falling back to eager")
-        return model_
```

Comment:
I have no objection to this change, but how come it's in this PR lol

Reply:
finger slipped lol